processing_chain/src/main/groovy/org/orbisgis/processingchain/PrepareBDTopo.groovy

```diff
@@ -0,0 +1,26 @@
+package org.orbisgis.processingchain
+
+import groovy.transform.BaseScript
+import org.orbisgis.bdtopo.BDTopoGISLayers
+import org.orbisgis.common.AbstractTablesInitialization
+import org.orbisgis.processmanager.ProcessMapper
+import org.orbisgis.processmanagerapi.IProcess
+
+@BaseScript ProcessingChain processingChain
+
+/**
+ *
+ * @return
+ */
```

Comment:
The javadoc is empty

Reply:
I fill it out. However, I wonder if the @param and @return parameters should be filled here. It would need to get into each of the processes to copy and paste the same name and definition. Would it be possible to get them automatically or should we copy and paste each of them (if we have several mappers using the same process, a modification of the process would need to modify all the mappers using it...)

Reply:
It would be great but quite tricky. I keep this idea in mind for a future improvement.

Reply:
Do not copy paste all parameters for each processes. Just describe the required parameters for the whole chain.